.meepctl-repocfg.yaml

```diff
@@ -82,6 +82,13 @@ repo:
         # OAuth k8s secret (data: client-id, secret)
         secret: meep-oauth-gitlab
 
+    # Default monitoring dashboards
```

Comment:
I was under the impression that some of the dashboards would be avail on AdvantEDGE
Ex. node exporter, k8s exporter, etc.

Reply:
Agreed. I moved the Node Exporter dashboard back this this repo.